modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/converter/ObjectEntryDTOConverter.java

```diff
@@ -153,6 +153,32 @@ private void _addNestedFields(
 		}
 	}
 
+	private void _addRelationshipNames(
+		Map<String, Object> map, ObjectField objectField, long objectEntryId,
+		String objectFieldName, ObjectRelationship objectRelationship,
+		Map<String, Serializable> values) {
+
+		map.put(objectFieldName, objectEntryId);
```

Comment:
Silly comment: I would put this at the end of the method to have all the additions to the map together (but sorted alphabetically 😁 )